djtriggers/tests/factories/triggers.py

```diff
@@ -0,0 +1,53 @@
+from django.utils import timezone
+from factory import DjangoModelFactory
+
+from djtriggers.models import Trigger
+from djtriggers.tests.models import NormalDummyTrigger, RaisesProcessErrorDummyTrigger, RaisesExceptionDummyTrigger
+
+
+class TriggerFactory(DjangoModelFactory):
+    class Meta:
+        model = Trigger
+
+    trigger_type = 'trigger_test'
+    source = 'tests'
+    date_received = timezone.now()
+    date_processed = None
+    process_after = None
+    number_of_tries = 0
+
+
+class NormalDummyTriggerFactory(DjangoModelFactory):
+    class Meta:
+        model = NormalDummyTrigger
+
+    trigger_type = 'normal_trigger_test'
+    source = 'tests'
+    date_received = timezone.now()
+    date_processed = None
+    process_after = None
+    number_of_tries = 0
+
+
+class RaisesExceptionDummyTriggerFactory(DjangoModelFactory):
+    class Meta:
+        model = RaisesExceptionDummyTrigger
+
+    trigger_type = 'failing_trigger_test'
+    source = 'tests'
+    date_received = timezone.now()
+    date_processed = None
+    process_after = None
+    number_of_tries = 0
+
+
+class RaisesProcessErrorDummyTriggerFactory(DjangoModelFactory):
```

Comment:
Im not sure what the intention of these error factories is. This one is not used in tests and afaik for the other one we could just as easily mock the process method to throw an error in the tests that require the trigger to fail